spell-check-custom-words.txt

```diff
@@ -50,6 +51,8 @@ Sway
 TAI
 TODO
 TypeScript
+UpgradePurpose
```

Comment:
Instead of adding UpgradePurpose and UpgradePurposeType to this file, you can just wrap them in backticks like `UpgradePurpose` so the spell checker will ignore them. Otherwise LGTM!